ui/src/selectors.js

```diff
@@ -51,13 +52,18 @@ export function selectLocale(state) {
   // request to metadata will be sent with unmodified Accept-Language headers
   // allowing the backend to perform language negotiation.
   const role = selectCurrentRole(state);
+  window.__localId__ = 'en';
+  
   if (role.locale) {
+    window.__localId__ = role.locale;
     return role.locale;
   }
   const { metadata } = state;
   if (metadata && metadata.app) {
+    window.__localId__ = metadata.app.locale;
     return metadata.app.locale;
```

Comment:
I think now with the FormattedRelativeTime component, we no longer need to set anything on the `window` ?

Reply:
Very true. I'll go through and remove this